src/main/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketPluginConfiguration.java

```diff
@@ -20,6 +21,9 @@
         "unused") // Stapler calls many of the methods via reflection (such as the setServerList)
 public class BitbucketPluginConfiguration extends GlobalConfiguration {
 
+    public static final String FORM_INVALID_C2A = " Please go back to the previous page and try again.";
```

Comment:
nitpick: Instead of whitespace at the start of the string, I'd prefer having whitespace when you actually do the concatenation